src/usage_metrics/raw/zenodo.py

```diff
@@ -0,0 +1,65 @@
+"""Extract data from Zenodo logs."""
```

Comment:
nit: might make sense to explain that this pulls logs from GCS, and there's a separate `save_zenodo_metrics.py` invocation in GHA that pulls everything to GCS. But maybe that whole architecture is explained elsewhere and I'm just learning this now 😅 .